modules/core-updates/8.0.md

```diff
@@ -77,6 +97,29 @@ The Bootstrap version used in Symfony-based Back office pages has been updated f
 * `Attribute`
   - Renamed to `ProductAttribute`.
 
+* `Cart`
+  - `replaceZeroByShopName()` - the parameter `$tr` [has been removed](https://github.com/PrestaShop/PrestaShop/pull/26913)
```

Comment:
The method is removed  

https://github.com/PrestaShop/PrestaShop/pull/27439

Reply:
I see. It's because we're going through the PRs from one to another, and at some point, something that we've documented is not valid anymore.

I'll make sure to check the `Changes` and remove what is not needed anymore. Thanks for the review ❤️

Reply:
You did a great job! I think for 9.0.0 we could be better on the BC documentation. Having a file with the PR number and the BC ( construct ,  method ... )  it would be easier to find the duplicates.